### PR TITLE
Updated Postgres doc

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Updated the Postgres doc

### CODE REVIEW

File: docs/integrations/sources/postgres.md [hunk 1]
```diff
@@ -1,263 +1,273 @@
...
+To create a replication slot called `airbyte_slot` using pgoutput, run:
 
-```text
+```
 SELECT pg_create_logical_replication_slot('airbyte_slot', 'pgoutput');
 ```
 
-If you would like to use `wal2json` plugin, please change `pgoutput` to `wal2json` value in the above query.
-
-#### <a name="initial-waiting-time"></a>6. [Optional] Set up initial waiting time
```
Comment (tuliren) [inline docs/integrations/sources/postgres.md:180]: @Amruta-Ranade, why is this entire section deleted? This information is important for the connector set up. In addition, the current set up page has a link to this section, which is broken now.

Screen Shot 2022-08-03 at 16 01 34

Comment (Amruta-Ranade) [reply docs/integrations/sources/postgres.md:180]: It isn't deleted..just restructured. If I'm not mistaken, the previous content didn't explicitly state the command to run. It does now: https://docs.airbyte.com/integrations/sources/postgres#step-3-create-replication-slot

Comment (tuliren) [reply docs/integrations/sources/postgres.md:180]: I am referring to `6. [Optional] Set up initial waiting time`, not `step 3`.

The doc about initial waiting time is no longer there in the latest master branch:
https://github.com/airbytehq/airbyte/blob/master/docs/integrations/sources/postgres.md

Comment (tuliren) [reply docs/integrations/sources/postgres.md:180]: Thank you~~

Comment (Amruta-Ranade) [reply docs/integrations/sources/postgres.md:180]: PR: https://github.com/airbytehq/airbyte/pull/15297

File: docs/integrations/sources/postgres.md [hunk 2]
```diff
@@ -1,263 +1,273 @@
...
+| Parameter  | Description  | Set value to |
+|------------|--------------|--------------|
+| wal_level  | Type of coding used within the Postgres write-ahead log  | logical |
+| max_wel_senders | The maximum number of processes used for handling WAL changes | Min: 1 |
+| max_replication_slots | The maximum number of replication slots that are allowed to stream WAL changes | 1 (if Airbyte is the only service reading subscribing to WAL changes. More than 1 if other services are also reading from the WAL) |
 
-We recommend using a `pgoutput` plugin as it is the standard logical decoding plugin in Postgres. In case the replication table contains a lot of big JSON blobs and table size exceeds 1 GB, we recommend using a `wal2json` instead. Please note that `wal2json` may require additional installation for Bare Metal, VMs \(EC2/GCE/etc\), Docker, etc. For more information read [wal2json documentation](https://github.com/eulerto/wal2json).
+To enable logical replication on AWS Postgres RDS or Aurora​:
 
-#### <a name="replication-slot"></a>4. Create replication slot
```
Comment (tuliren) [inline docs/integrations/sources/postgres.md:148]: @Amruta-Ranade, the internal anchors were deleted here. They are actually referenced in the `spec.json` in some of the field descriptions. I am adding them back in https://github.com/airbytehq/airbyte/pull/15382.

**Comment by Amruta-Ranade** [comment]
Thank you!!

**Pull request merged**